CHANGES.rst

```diff
@@ -6,6 +6,9 @@ There's a frood who really knows where his towel is.
 2.2.1 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
+- An auxiliary variable is used to refer to the selected object, in this way it is always ensured that the events are correctly applied to the indicated object. (fixes `#871 <https://github.com/collective/collective.cover/issues/871>`_ and fixes `#859 <https://github.com/collective/collective.cover/issues/859>`_).
```

Comment:
In changes, you must say what has been fixed. You can even say how you fixed, but you must also say what was fixed.

Please enter an entry for issue.